octobot_trading/exchanges/traders/trader.py

```diff
@@ -389,7 +390,9 @@ async def cancel_order_with_id(self, order_id, emit_trading_signals=False,
     async def cancel_open_orders(self, symbol, cancel_loaded_orders=True, side=None,
                                  emit_trading_signals=False,
                                  wait_for_cancelling=True,
-                                 cancelling_timeout=octobot_trading.constants.INDIVIDUAL_ORDER_SYNC_TIMEOUT
+                                 cancelling_timeout=octobot_trading.constants.INDIVIDUAL_ORDER_SYNC_TIMEOUT, 
+                                 since: typing.Union[int, float] = -1, 
```

Comment:
We try to avoid using magic numbers, can you create a constant for default since and until values please?

Reply:
I replaced it with a constant.